context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+ContextFree is for people who want to create designs with lots of similar 
+geometric images that rely on symmetry.
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to create symmetrical geometric patterns concisely because 
+ContextFree uses recursion to build them in just a few lines of code.
 
 ## What is hard to do in this language? Why is it hard?
 
+Displaying words and sentences can be difficult in ContextFree because this 
+design cannot take advantage of recursion. For example, in order to create the 
+Welcome demo, you need to write rules for each unique letter and then place 
+each letter in the correct position. Imagine how much code would be required to 
+write a Chinese phrase, with so many characters and strokes.
```

Comment:
Didn't even think about the Chinese characters part. Although I'd argue that after you made rules for each stroke, you can put together characters pretty easily (adjusting for size and angle). However, I have to agree that making the rules for each stroke or English letter or what have you is slow and hard to do.